src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -181,8 +184,14 @@ bool TvgSaver::saveEncoding(const std::string& path)
     memcpy(uncompressed, &compressedSizeBits, TVG_HEADER_COMPRESSED_SIZE_BITS);
 
     //Good optimization, flush to file.
+#ifdef __clang__
```

Comment:
Not negative at this but it would be better to split to a function for removing duplicates.

static FILE* _fopen(...)
{
#ifdef __clang__
#else
#endif
}

Reply:
done